### DISCUSSION TITLE
How to use HasSortableRelations?

### DISCUSSION THREAD

**Discussion by dotch2010** [wintercms/winter#756] [Q&A]
Please show an example of using HasSortableRelations. I can't figure out how to sort my relation

**Comment by bennothommo** [comment]
Sorry for the delay in responding to this one, @dotch2010.

We don't have an example yet - this trait was brought in mainly to support this PR (https://github.com/wintercms/winter/pull/553) but since that's not merged yet, there's no garuntee it works.

**Comment by jsebaf** [comment]
Hi! 
I see that the PR mentioned is already merged. Can you give me any hints on how it should be used?

Thanks!

**Reply by jaxwilko** [reply]
Hi @jsebaf, the PR was closed without merge
